packages/components/src/components/notification/notification.stories.ts

```diff
@@ -189,22 +216,13 @@ export const ToastNotification = {
         });
       </script>
     `;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const button = canvasElement.querySelector('#top-right');
-    await userEvent.click(button!);
   }
 };
 
 /**
  * Display a toast notification positioned at the bottom-center of the screen by setting the `toastStack` attribute to `bottom-center`. Click on the **Show code** button to see the JavaScript code responsible for generating the toast notification.
```

Comment:
In figma de have both bottom and top notifications under "Toast Notification". Is this possible in CODE?